low-level/files/opt/seagate/sspl/setup/sspl_reset.py

```diff
@@ -50,11 +53,9 @@ def process(self):
         shutil.rmtree(DATA_PATH, ignore_errors=True)
 
         # Clear log data from log files and delete 'log.gz' files
-        Reset.reset_log_files('.log', f"/var/log/{PRODUCT_FAMILY}/sspl/")
-        Reset.reset_log_files('.log', f"/var/log/{PRODUCT_FAMILY}/iem/")
-        Reset.reset_log_files('.log.gz',
-                              f"/var/log/{PRODUCT_FAMILY}/sspl/",
-                              del_file=True)
-        Reset.reset_log_files('.log.gz',
-                              f"/var/log/{PRODUCT_FAMILY}/iem/",
-                              del_file=True)
+        Utility.reset_log_files(f"/var/log/{PRODUCT_FAMILY}/sspl/", '.log')
```

Comment:
sspl.log & iem.log also needs to be deleted right, thats what we decided for pre-factory right !?

Reply:
yes, for pre-factory we are deleting log files.